packages/admin/cms-admin/src/userPermissions/user/permissions/selectScopesDialog/SelectScopesDialog.tsx

```diff
@@ -0,0 +1,135 @@
+        data?.availableContentScopes || [],
+        intl,
+    );
+
+    return (
+        <SaveBoundary
+            onAfterSave={() => {
+                onClose();
+            }}
+        >
+            <Dialog open={open} maxWidth="lg">
+                <DialogTitle>
+                    <FormattedMessage id="comet.userScopes.dialog.title" defaultMessage="Select scopes" />
+                </DialogTitle>
+                <FinalForm<FormValues>
+                    apiRef={formApiRef}
+                    subscription={{ values: true }}
+                    mode="edit"
+                    onSubmit={submit}
+                    onAfterSubmit={() => null}
+                    initialValues={{
+                        contentScopes: data.userContentScopes.map((cs) => JSON.stringify(cs)),
+                    }}
+                >
+                    <Field name="contentScopes" fullWidth>
+                        {(props) => {
+                            return (
+                                <DataGrid
+                                    autoHeight={true}
+                                    rows={
+                                        data.availableContentScopes.filter((obj) => !Object.values(obj).every((value) => value === undefined)) ?? []
+                                    }
+                                    columns={columns}
+                                    rowCount={data?.availableContentScopes.length ?? 0}
+                                    loading={false}
+                                    getRowHeight={() => "auto"}
+                                    getRowId={(row) => JSON.stringify(row)}
+                                    isRowSelectable={(params) => {
+                                        return !data.userContentScopesSkipManual.some(
+                                            (cs: ContentScope) => JSON.stringify(cs) === JSON.stringify(params.row),
+                                        );
+                                    }}
+                                    checkboxSelection
+                                    sx={{ "&.MuiDataGrid-root .MuiDataGrid-cell": { py: "8px" }, width: "100%" }}
+                                    selectionModel={props.input.value}
+                                    onSelectionModelChange={(selectionModel) => {
+                                        props.input.onChange(selectionModel.map((id) => String(id)));
+                                    }}
+                                    components={{
+                                        Toolbar: SelectScopesGridToolbar,
+                                    }}
+                                />
+                            );
+                        }}
+                    </Field>
+                </FinalForm>
+                <DialogActions>
+                    <CancelButton onClick={onClose}>
+                        <FormattedMessage id="comet.userScopes.close" defaultMessage="Close" />
```

Comment:
We have a generic message for this: https://github.com/vivid-planet/comet/blob/main/packages/admin/admin/src/messages.ts#L49

Reply:
okay :) didnt know that :) will change it :)